docs/sphinx/using/backends/dynamics.rst

```diff
@@ -184,6 +214,13 @@ The following code sets up the problem
         :start-after: [Begin Hamiltonian]
         :end-before: [End Hamiltonian]
 
+.. tab:: C++
+
+  .. literalinclude:: ../../snippets/cpp/using/backends/dynamics.cpp
+        :language: cpp
+        :start-after: [Begin Hamiltonian]
+        :end-before: [End Hamiltonian]
+
 2. Time-dependent operator
 
 We can also construct a time-dependent operator from a function that returns a complex matrix representing the time dynamics of 
```

Comment:
There are 3 more Python tabs in this file - do they have a C++ equivalent?
Also, for the table of integrators at the bottom, I believe these are just for python? Which C++ integrators should we document?

Reply:
Indeed, I think we should have equivalent C++ code snippets for the three boxes below demonstrating how to define time-dependent operators.

For the integrator section, as this is now a unified Python/C++ page, we should add some text to denote that the current table applies to Python only and another paragraph about C++ (only RungeKutta is provided).